packages/fluentui/react-northstar/test/specs/components/Datepicker/validateDate-test.tsx

```diff
@@ -1,5 +1,5 @@
 import { validateDate } from 'src/components/Datepicker/validateDate';
-import { DEFAULT_CALENDAR_STRINGS, IRestrictedDatesOptions } from '@fluentui/date-time-utilities';
+import { DEFAULT_CALENDAR_STRINGS, IRestrictedDatesOptions } from 'src/utils/date-time-utilities';
```

Comment:
The `src` alias is considered deprecated:
https://github.com/microsoft/fluentui/blob/50c2a17463c06292662904065cbeba1ee05122a1/packages/fluentui/react-northstar/jest.config.js#L6-L8
Use relative path.

Reply:
Trying to use relative path I get `Cannot find module '../../utils/date-time-utils'.ts(2307)`

Reply:
In this case you'd need to use ../../../**src**/utils/date-time-utils